client/components/mma/shared/ProductInfoTableV2.tsx

```diff
@@ -0,0 +1,83 @@
+import { css } from '@emotion/react';
+import {
+	from,
+	palette,
+	space,
+	textSans,
+	until,
+} from '@guardian/source-foundations';
+import { parseDate } from '@/shared/dates';
+import type { ProductDetail } from '@/shared/productResponse';
+
+export interface ProductInfoTableProps {
+	productDetail: ProductDetail;
+}
+
+export const ProductInfoTableV2 = (props: ProductInfoTableProps) => {
+	const valueBoxCss = () => {
+		return css`
+			background-color: ${palette.neutral[97]};
+			display: flex;
+			width: 803px;
+			${until.tablet} {
+				width: 351px;
```

Comment:
not sure about these widths, feel like there may be a nicer way to do it

Reply:
![image](https://github.com/guardian/manage-frontend/assets/114918544/2882bd33-3496-456e-ab3b-ac819b26ac62)
is the width going past the edge?